config/main.py

```diff
@@ -23,6 +23,7 @@
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help', '-?'])
 
+SONIC_GENERATED_SERVICE_PATH = '/etc/sonic/generated_services.conf'
```

Comment:
Changes looks ok. 
And that this file  '/etc/sonic/generated_services.conf' should be present in all platforms.